lib/angular-rails-templates/engine.rb

```diff
@@ -37,7 +37,7 @@ class Engine < ::Rails::Engine
 
       # Sprockets Cache Busting
       # If ART's version or settings change, expire and recompile all assets
-      hash_digest = defined?(ActiveSupport::Digest) ? ActiveSupport::Digest : Digest::MD5
+      hash_digest = app.config.active_support.hash_digest_class
```

Comment:
This ends up being `nil` because it's only set in the Rails 5.2 defaults:
https://github.com/rails/rails/blob/00cc4ff0259c0185fe08baadaa40e63ea2534f6e/railties/lib/rails/application/configuration.rb#L150

But the demo config doesn't call `load_defaults`. Also, if you do that, then there is still CI to test for Rails 5.0. Real world applications also may not call it. In other words, you can't assume there is a hash digest set and need fallback code.

I think this should be safe:

```suggestion
      hash_digest = app.config.active_support.hash_digest_class || ActiveSupport::Digest.hash_digest_class
```

Reply:
I updated with your idea and then further modified it to account for Rails 5.0 support and it lacking the Digest class in ActiveSupport.